### PR TITLE
Cortx 28178 corio old log cleanup/backup

### PR DESCRIPTION
**Pull request by DhobaleVishal**
# Problem Statement
-  Create backup of old log/latest & reports.

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed
-  [ ] Attach test execution logs

# Review Checklist 
  Checklist for Author
-  [x] JIRA number/GitHub Issue added to PR
-  [x] PR is self reviewed
-  [x] Jira and state/status is updated and JIRA is updated with PR link
-  [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe

### CODE REVIEW

**Comment by VikasKumarC** [comment]
Hi @DhobaleVishal  as discussed. Please add log for Object size and No. of sessions in test scripts.

**Comment by DhobaleVishal** [comment]
> Hi @DhobaleVishal as discussed. Please add log for Object size and No. of sessions in test scripts.

@VikasKumarC  I will create seperate PR for that.

File: src/commons/utils/corio_utils.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import os
+import logging
+import glob
+from datetime import datetime
+
+LOGGER = logging.getLogger(__name__)
+
+
+def cleanup():
```
Comment (ketanarlulkar) [inline src/commons/utils/corio_utils.py]: rename to log cleanup, we need to have one more cleanup function to delete object files.

File: corio.py [hunk 2]
```diff
@@ -190,26 +192,26 @@ def setup_environment():
...
     Log execution status into log file.
 
     :param parsed_input: Dict for all the input yaml files
     :param corio_start_time: Start time for main process
     :param test_failed: Reason for failure is any
     """
-    LOGGER.info("Logging current status to corio_status.log")
-    with open('corio_status.log', 'w') as status_file:
+    status_fpath = os.path.join(
+        os.getcwd(), "reports", f"corio_status_{datetime.now().strftime('%d_%m_%Y_%H_%M_%S')}.log")
```
Comment (ketanarlulkar) [inline corio.py]: Use `DT_STRING` instead of datetime.now(), so both report and log will have similar name

**Review by ketanarlulkar — CHANGES_REQUESTED** [review]

File: corio.py (hunk 2 above)
Comment (priyankaborawake) [reply corio.py]: we cannot use datetime.now() as only one file should be created for status log and the same is being updated every half hour.

**Review by ketanarlulkar — APPROVED** [review]

**Review by priyankaborawake — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**